### DISCUSSION TITLE
All editions: option to reset to the vanilla theme

### DISCUSSION THREAD

**Discussion by ding0z** [SpiralLinux/SpiralLinux-project#119] [Q&A] [answered]
Hi
Would it be possible to include an option to reset the themes and icons to the original ones included with a particular DE (or is there a way to easily do it after the installation)?

I love Spiral Linux but the chosen wallpaper icons are a bit too old style 🤭

An alternative would be to chose between several themes available but I believe it would require more work. 

Anyway thanks for the fantastic job.

**Comment by geckolinux** [comment, marked as answer]
Hi there, really glad that you're enjoying SpiralLinux.

It's super easy to reset the entire desktop configuration (panel layout, application defaults, etc.), just delete `.config` in your `/home/username` directory.

If it's just the theme you want to change, do be aware that the default Debian theme is truly terrible. If you want to change the theme it would make more sense to just install other themes and set them in the desktop environment's settings. But keep in mind that the current theme is carefully selected because of things like missing icons and wrong panel tray icons in other icon sets.

Hope this helps!

**Reply by ding0z** [reply]
Thanks for the answer!
